node/src/components/consensus/era_supervisor.rs

```diff
@@ -188,19 +190,24 @@ where
     fn booking_block_height(&self, era_id: EraId) -> u64 {
         // The booking block for era N is the last block of era N - AUCTION_DELAY - 1
         // To find it, we get the start height of era N - AUCTION_DELAY and subtract 1
-        let after_booking_era_id =
-            EraId(era_id.0.saturating_sub(self.protocol_config.auction_delay));
+        let after_booking_era_id = EraId(
+            era_id
+                .0
+                .saturating_sub(self.protocol_config.auction_delay + self.initial_era.0)
+                + self.initial_era.0,
```

Comment:
Maybe `.max(self.initial_era.0)` would be more readable than subtracting and adding again.